codegenerator/cli/templates/static/codegen/src/eventFetching/FetchState.res

```diff
@@ -15,14 +15,9 @@ type blockNumberAndLogIndex = {blockNumber: int, logIndex: int}
 
 type status = {mutable fetchingStateId: option<int>}
 
-type partitionKind =
-  | Wildcard
-  | Normal({
-      contractAddressMapping: ContractAddressingMap.mapping,
-      //Used to prune dynamic contract registrations in the event
-      //of a rollback.
-      dynamicContracts: array<TablesStatic.DynamicContractRegistry.t>,
-    })
+type selection =
+  | Wildcard({})
+  | Normal({})
```

Comment:
What is the point of payloads in these variant constructors?

Reply:
So, they are objects and can be used as WeakMap keys.

Reply:
In the next versions, they will also contain a list of specific events. You can see how it's done for wildcard in the next PR.